Userland/Libraries/LibProtocol/WebSocket.cpp

```diff
@@ -20,6 +20,11 @@ WebSocket::ReadyState WebSocket::ready_state()
     return (WebSocket::ReadyState)m_client->ready_state({}, *this);
 }
 
+DeprecatedString WebSocket::subprotocol_in_use()
+{
+    return (DeprecatedString)m_client->subprotocol_in_use({}, *this);
```

Comment:
Using a C-style cast to a C++ class is a little weird. Is this cast even needed?  I would think it'd be implicit.

Anyway, if it is needed do something a bit more C++-y like `DeprecatedString { m_client->subprotocol_in_use({}, *this) }`.